TAO/TAO_IDL/util/utl_err.cpp

```diff
@@ -185,9 +185,12 @@ error_string (UTL_Error::ErrorCode c)
     case UTL_Error::EIDL_KEYWORD_WARNING:
       return "Warning - spelling differs from IDL keyword only in case: ";
     case UTL_Error::EIDL_ANONYMOUS_ERROR:
-      return "Error: anonymous types are deprecated by OMG spec";
+      return "anonymous types require the IDL version to be 4 or later or must "
```

Comment:
Shouldn't be here Error/Warning be used in the message?

Reply:
"Warning" and "Error"  are already printed by `warning_header` and `error_header` functions respectively.

Reply:
Ok